src/Lykke.Job.OrderBooksCacheProvider/Modules/JobModule.cs

```diff
@@ -87,5 +73,26 @@ private void RegisterPeriodicalHandlers(ContainerBuilder builder)
                 .AutoActivate()
                 .SingleInstance();
         }
+
+        private void RegisterRedis(ContainerBuilder builder)
+        {
+            System.Threading.ThreadPool.SetMinThreads(100, 100);
+            var options = ConfigurationOptions.Parse(_settings.CacheSettings.RedisConfiguration);
+            options.ReconnectRetryPolicy = new ExponentialRetry(3000, 15000);
+            options.ClientName = "Lykke.Job.OrderBooksCacheProvider";
+
+            var redis = ConnectionMultiplexer.Connect(options);
+
+            builder.RegisterInstance(redis).SingleInstance();
+            builder.Register(
+                c =>
+                    c.Resolve<ConnectionMultiplexer>()
+                        .GetServer(redis.GetEndPoints()[0]));
+
+            builder.Register(
```

Comment:
This is not required since:
The object returned from GetDatabase is a cheap pass-thru object, and does not need to be stored.

Reply:
Fixed.